Resources/Private/Fusion/Prototypes/TrackingCode.GA.fusion

```diff
@@ -0,0 +1,20 @@
+prototype(Neos.GoogleAnalytics:TrackingCode.GA) < prototype(Neos.GoogleAnalytics:TrackingCode.Base) {
+    trackingId = ${TrackingConfiguration.setting(site, 'analytics.id')}
+    @if.trackingId =${!String.isBlank(this.trackingId)}
```

Comment:
Should we prevent users from both rendering the GTM and the GA code?

Reply:
Yes, I think we should.